utils/githooks/pre-commit.d/10-update-copyright

```diff
@@ -39,20 +45,16 @@ targets=(
     '.env'
 )
 
+if [ -z "$files" ]; then
+    files=$(git diff "$TARGET" --cached --diff-filter=AM --name-only -- "${targets[@]}")
+else
+    echo "  Checking against custom files"
+fi
```

Comment:
I don't see where `files` would be set. Presumable in someone's ENV? Maybe we could just remove this handling now